slm_porosity/heat_transfer.f90

```diff
@@ -55,13 +55,18 @@ MODULE user_case
   REAL(pr) :: fusion_heat
 
   ! bed parameters
-  REAL(pr) :: laser_power
-  REAL(pr) :: scanning_speed
   REAL(pr) :: initial_porosity
   REAL(pr) :: convective_transfer
   REAL(pr) :: radiative_transfer
   REAL(pr) :: absolute_temperature
 
+  ! laser parameters
+  REAL(pr) :: laser_power
+  REAL(pr) :: scanning_speed
+  REAL(pr) :: laser_beam_length
+  REAL(pr) :: laser_beam_width
```

Comment:
There is a point about normalization and introducing inclination_angle. Math must be rechecked cause there could be some significant tail loosing numbers with increasing of 'laser_beam_length' and 'laser_beam_width' which would lead to loosing of energy.  On the other hand what is laser radius? Half-width of the gaussian? Maybe tail loosing is ok for our case.

Reply:
I suggested `laser_beam_length=laser_beam_width`. If you want to implement inclination, create a new issue. Generally, create issue for every your idea. After, you are able to prioritize your issues. This is a classical development system.